src/viur/core/secret.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from google.cloud import secretmanager
+from viur.core.config import conf
+
+"""
+This module provides utiliy functions for accessing values stored in the Google Cloud Secret Manager.
+"""
+
+# Global secret manager client instance
+__client = secretmanager.SecretManagerServiceClient()
+
+
+def get(secret: str, version: str = "latest") -> str:
+    """
+    Retrieves a secret stored in Google Cloud Secret Manager for use within the application.
+
+    Add a secret online under https://console.cloud.google.com/security/secret-manager.
+    Service accounts requires the role "Secret Manager Secret Accessor" in IAM.
+    """
+    name = f"""projects/{conf["viur.instance.project_id"]}/secrets/{secret}/versions/{version}""".replace(".", "_")
```

Comment:
What's the reason for the `replace(".", "_")`? Dots (`.`) are not allowed in secret names, so just don't use them!